Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -199,13 +200,31 @@ extension SettingsAccountViewController: UITableViewDelegate {
   }
 
   func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-    return 0.1
+
+    let (userHasPassword, _) = self.viewModel.outputs.userHasPasswordAndEmail
+    guard section == SettingsAccountSectionType.createPassword.rawValue && !userHasPassword else {
+      return 0.1
+    }
+    return Styles.grid(9)
   }
 
   func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
     return tableView.dequeueReusableHeaderFooterView(withIdentifier: Nib.SettingsHeaderView.rawValue)
   }
 
+  func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+
+    let (userHasPassword, email) = self.viewModel.outputs.userHasPasswordAndEmail
+    guard section == SettingsAccountSectionType.createPassword.rawValue && !userHasPassword else {
+      return nil
+    }
+
+    let footerView = tableView.dequeueReusableHeaderFooterView(
```

Comment:
So for the optional comment I had bellow, this is where we would not display a section if email from the signal was `nil`